specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-07-01-preview/inference.yaml

```diff
@@ -0,0 +1,7823 @@
+        - first_id
+        - last_id
+        - has_more
+
+    messageContentImageFileObject:
+      title: Image file
+      type: object
+      description: References an image file in the content of a message.
+      properties:
+        type:
+          description: Always `image_file`.
+          type: string
+          enum:
+            - image_file
+          x-ms-enum:
+            name: MessageContentImageFileObjectType
+            modelAsString: true
+            values:
+              - value: image_file
+                description: The message content image file type
+        image_file:
+          type: object
+          properties:
+            file_id:
+              description: The file ID of the image in the message content.
+              type: string
+          required:
+            - file_id
+      required:
+        - type
+        - image_file
+
+    messageContentTextObject:
+      title: Text
+      type: object
+      description: The text content that is part of a message.
+      properties:
+        type:
+          description: Always `text`.
+          type: string
+          enum:
+            - text
+          x-ms-enum:
+            name: messageContentTextObjectType
+            modelAsString: true
+            values:
+              - value: text
+                description: The message content text Object type
+        text:
+          type: object
+          properties:
+            value:
+              description: The data that makes up the text.
+              type: string
+            annotations:
+              type: array
+              items:
+                oneOf:
+                  - $ref: '#/components/schemas/messageContentTextAnnotationsFileCitationObject'
+                  - $ref: '#/components/schemas/messageContentTextAnnotationsFilePathObject'
+          required:
+            - value
+            - annotations
+      required:
+        - type
+        - text
+
+    messageContentTextAnnotationsFileCitationObject:
+      title: File citation
+      type: object
+      description: A citation within the message that points to a specific quote from a specific File associated with the assistant or the message. Generated when the assistant uses the "retrieval" tool to search files.
+      properties:
+        type:
+          description: Always `file_citation`.
+          type: string
+          enum:
+            - file_citation
+          x-ms-enum:
+            name: FileCitationObjectType
+            modelAsString: true
+            values:
+              - value: file_citation
+                description: The file citation object type
+        text:
+          description: The text in the message content that needs to be replaced.
+          type: string
+        file_citation:
+          type: object
+          properties:
+            file_id:
+              description: The ID of the specific File the citation is from.
+              type: string
+            quote:
```

Comment:
This isn't blocking and can be deferred: OpenAI removed "quote" from their citations a while back with this [this commit](https://github.com/openai/openai-openapi/commit/dd73070b1d507645d24c249a63ebebd3ec38c0cb) -- we should likely match the parity surface when it's convenient and verifiable.

Reply:
Merged the change made by Shruti to remove it.